tests/baselines/reference/privateNamesUnique-2.types

```diff
@@ -65,12 +65,12 @@ const b = new B();
 a.copy(b); // error
 >a.copy(b) : void
 >          : ^^^^
->a.copy : (other: import(} fro).Foo) => void
->       : ^     ^^                 ^^^^^^^^^
+>a.copy : (other: import("b").Foo) => void
+>       : ^     ^^                     ^^^^^^^^^
 >a : A
 >  : ^
->copy : (other: import(} fro).Foo) => void

```

Comment:
That we already had a test demonstrating the issue is... yeah, reviewing big baseline diffs is hard - easy to miss the *one* test that's still off. And that's why we try and break it down where we can. 🤷‍♂️

Reply:
Oops!